vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -2180,12 +2181,29 @@ func (e *CaminoStandardTxExecutor) AddressStateTx(tx *txs.AddressStateTx) error
 		return err
 	}
 
-	roles := as.AddressStateEmpty
-	creds := e.Tx.Creds
-
 	chainTime := e.State.GetTimestamp()
+	statesBit := tx.StateBit.ToAddressState()
```

Comment:
Can we rename this to something like `txAddressState`? `statesBit` is somewhat confusing.